index.html

```diff
@@ -5,96 +5,125 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel = "stylesheet"
+  type = "text/css"
+  href = "./stylesheets/style.css" />
   <script src="https://use.fontawesome.com/1fb0eb3889.js"></script>
   <title>Where work happens | Slack</title>
 </head>
 
 <body>
+  <div id="navbar">
   <nav>
-    <img src="./images/slack-logo.png" alt="slack_logo">
-    <ul>
+    <img id="logo" src="./images/slack-logo.png" alt="slack_logo">
+    <ul id="navlist">
       <li>Why Slack?</li>
       <li>Pricing</li>
       <li>About Us</li>
       <li>Your Workspaces</li>
     </ul>
   </nav>
+ </div>
+  <div id="header-div">
+    <div class="bigger-box">
   <header>
-    <img src="./images/home_talking.png" alt="home_work_happening">
-    <h1>Where Work Happens</h1>
-    <p>When your team needs to kick off a project, hire a new employee, deploy some code, review a sales contract, finalize next year's budget, measure an A/B test, plan your next office opening, and more, Slack has you covered.</p>
-    <button type="button" name="button">GET STARTED</button>
-    <p>Already using Slack? <a href="#">Sign in</a></p>
+    <div><h1>Where Work Happens</h1>
+    <p id="not-center">When your team needs to kick off a project, hire a new employee, deploy some code, review a sales contract, finalize next year's budget, measure an A/B test, plan your next office opening, and more, Slack has you covered.</p>
+  </div>
+  <div>
+    <input type="text" name="fname">
+    <button id="buttonTop" type="button" name="button">GET STARTED</button>
+    <p class="text-center">Already using Slack? <a href="#">Sign in</a></p>
+  </div>
   </header>
+</div>
   <main>
-    <div>
-      <h2>You´re in good company</h2>
+    <div class="bigger-box">
+      <img src="./images/home_talking.png" alt="home_work_happening">
+    </div>
+  </div>
+      <div class="greyboxes">
```

Comment:
try to keep your classes in html written in kebab case. Also try to name them relevant to the page rather than the style.